mobile/lib/main.dart

```diff
@@ -396,3 +399,26 @@ class ScaffoldWithNavBar extends StatelessWidget {
     }
   }
 }
+
+Future<void> logAppSettings(bridge.Config config) async {
+  String commit = const String.fromEnvironment('COMMIT');
+  if (commit.isNotEmpty) {
+    FLog.info(text: "Built on commit: $commit");
+  }
+
+  String branch = const String.fromEnvironment('BRANCH');
+  if (branch.isNotEmpty) {
+    FLog.info(text: "Built on branch: $branch");
+  }
+
+  PackageInfo packageInfo = await PackageInfo.fromPlatform();
+  FLog.info(text: "Build number: ${packageInfo.buildNumber}");
+  FLog.info(text: "Build version: ${packageInfo.version}");
+
+  FLog.info(text: "Network: ${config.network}");
+  FLog.info(text: "Esplora endpoint: ${config.esploraEndpoint}");
+  FLog.info(text: "Coordinator: ${config.coordinatorPubkey}@${config.host}:${config.p2PPort}");
```

Comment:
🔧 Can you add oracle endpoint and pubkey?

Reply:
Good idea!